tools/ci_build/github/azure-pipelines/stages/py-cuda-publishing-stage.yml

```diff
@@ -0,0 +1,37 @@
+parameters:
+  - name: nightly
+    type: string
+    default: '1'
+  - name: repository_url
+    type: string
+    default: 'https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-cuda-12-nightly/pypi/upload'
+
+stages:
+  - stage: Python_Publishing
+    dependsOn: [ Python_Packaging ]
+    variables:
+      - name: artifact_feed
+        ${{ if ne(parameters.nightly, '1') }}:
+          value: onnxruntime-cuda-12
+        ${{ else }}:
+          value: ort-cuda-12-nightly
+    jobs:
+      - job:
+        steps:
+          - task: DownloadPipelineArtifact@2
+            inputs:
+              artifact: 'onnxruntime-gpu'
+              targetPath: '$(Build.SourcesDirectory)'
+            displayName: 'Download Pipeline Artifact - onnxruntime-gpu'
+          - task: UsePythonVersion@0
+            displayName: 'Use Python 3.x'
+          - script: 'pip install twine==3.4.2'
+            displayName: 'Install Twine'
+          - task: TwineAuthenticate@1
```

Comment:
Use a service connection for this.

https://learn.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml#python-package-upload-service-connection

Reply:
Like this:

```yaml
steps:
- task: TwineAuthenticate@1
  displayName: 'Twine Authenticate '
  inputs:
    artifactFeed: 'PublicPackages/ORT-Nightly'

- script: ' find . -type f -name ''*.whl'' | xargs -I % python -m twine upload --skip-existing % -r ORT-Nightly --config-file $(PYPIRC_PATH)'
  workingDirectory: '$(System.DefaultWorkingDirectory)/training cuda wheels'
  displayName: 'Uploading wheels'
```